rules/cross-platform/defense_evasion_deleting_websvr_access_logs.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/11/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies deletion of webserver access logs. This may indicate an attempt to evade detection or destroy forensic
```

Comment:
```suggestion
Identifies the deletion of WebServer access logs. This may indicate an attempt to evade detection or destroy forensic
evidence on a system.
```